guides/common/attributes-base.adoc

```diff
@@ -18,13 +18,13 @@
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 // For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
 :RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-6-beta-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.8-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.9-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.9-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.9-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-server-7-satellite-capsule-6-beta-rpms
+:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-server-7-satellite-tools-6-beta-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-server-7-satellite-maintenance-6-rpms
 // Do not update the puppet4 repo versions. They must stay at 6.3.
-:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
-:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-server-7-satellite-6.3-puppet4-rpms
+:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-server-7-satellite-capsule-6.3-puppet4-rpms
```

Comment:
This hunk is incorrect, keep it unchanged.

```
# subscription-manager repos --list | grep puppet4 | grep ID:
Repo ID:   rhel-7-server-satellite-capsule-6.3-puppet4-debug-rpms
Repo ID:   rhel-7-server-aus-satellite-tools-6.3-puppet4-source-rpms
Repo ID:   rhel-7-server-satellite-tools-6.3-puppet4-rpms
Repo ID:   rhel-7-server-satellite-capsule-6.3-puppet4-source-rpms
Repo ID:   rhel-7-server-satellite-6.3-puppet4-debug-rpms
Repo ID:   rhel-7-server-satellite-6.3-puppet4-rpms
Repo ID:   rhel-7-server-eus-satellite-tools-6.3-puppet4-source-rpms
Repo ID:   rhel-7-server-eus-satellite-tools-6.3-puppet4-debug-rpms
Repo ID:   rhel-7-server-e4s-satellite-tools-6.3-puppet4-debug-rpms
Repo ID:   rhel-7-server-aus-satellite-tools-6.3-puppet4-rpms
Repo ID:   rhel-7-server-satellite-6.3-puppet4-source-rpms
Repo ID:   rhel-7-server-aus-satellite-tools-6.3-puppet4-debug-rpms
Repo ID:   rhel-7-server-eus-satellite-tools-6.3-puppet4-rpms
Repo ID:   rhel-7-server-satellite-tools-6.3-puppet4-source-rpms
Repo ID:   rhel-7-server-e4s-satellite-tools-6.3-puppet4-source-rpms
Repo ID:   rhel-7-server-e4s-satellite-tools-6.3-puppet4-rpms
Repo ID:   rhel-7-server-satellite-tools-6.3-puppet4-debug-rpms
Repo ID:   rhel-7-server-satellite-capsule-6.3-puppet4-rpms
```

Reply:
done in https://github.com/theforeman/foreman-documentation/pull/734/commits/110057dcbd530561229e594bee1ef44a2b1d45c0